app/views/layouts/darkswarm.html.haml

```diff
@@ -37,6 +37,7 @@
           #footer
     %loading
 
+    = render "layouts/bugsnag_js"
```

Comment:
I just read on bugsnag
"Include Bugsnag via a <script> tag in the <head> of your HTML:" 
"It’s best to initialize Bugsnag before any other JavaScript so it can catch as many errors as possible."

@mkllnk any specific reason to add it to the body?
maybe we want to create a new PR to move it to the head in a later release?

Reply:
I just put it above all the other Javascript which we load at the bottom. Can you see anything else being loaded before? If not, it won't make a difference to load it earlier.

Reply:
ok, I was reading a bit about this and I think we are ok here. There's no difference between head and body for js scripts. Only the order matters and async or no async. In these cases I think we cant use async.
All good.